src/pineko/scaffold.py

```diff
@@ -0,0 +1,59 @@
+"""Tools related to generation and managing of a pineko project."""
+
+import os
+import pathlib
+
+from .configs import needed_keys
+
+
+def set_up_project(configs):
+    """Set up all the folders spelled up in the configs dictionary.
+
+    Parameters
+    ----------
+    configs : dict
+        configs dictionary containing all the paths to be setted up
+    """
+    for path in configs["paths"]:
+        if path == "root":
+            continue
+        if isinstance(configs["paths"][path], pathlib.Path):
+            if configs["paths"][path].suffix == ".yaml":
+                continue
+            os.makedirs(configs["paths"][path], exist_ok=True)
```

Comment:
`pathlib` should reimplement `os` - e.g. here you can do instead https://docs.python.org/3/library/pathlib.html?highlight=pathlib#pathlib.Path.mkdir